fastlane/lanes/release.rb

```diff
@@ -262,18 +332,40 @@ def trigger_buildkite_release_build(branch:, beta:)
   buildkite_annotate(style: 'info', context: 'trigger-release-build', message: message)
 end
 
-def create_release_backmerge_pr(version_to_merge:, next_version:)
+def create_backmerge_pr!
+  pr_urls = create_backmerge_prs
+
+  return pr_urls unless pr_urls > 1
+
+  backmerge_error_message = UI.user_error! <<~ERROR
+    Unexpectedly opened more than one backmerge pull request. URLs:
+    #{pr_urls.map { |url| "- #{url}" }.join("\n")}
+  ERROR
+  buildkite_annotate(style: 'error', context: 'error-creating-backmerge', message: backmerge_error_message) if is_ci
+  UI.user_error!(backmerge_error_message)
+end
+
+# Notice the plural in the name.
+# The action this method calls may create multiple backmerge PRs, depending on how many releas branches with version greater than the source are in the remote.
+def create_backmerge_prs
+  version = release_version_current
+
   create_release_backmerge_pull_request(
     repository: GITHUB_REPO,
-    source_branch: release_branch_name(release_version: version_to_merge),
+    source_branch: release_branch_name(release_version: version),
     labels: ['Releases'],
-    milestone_title: next_version
+    milestone_title: release_version_next
   )
 rescue StandardError => e
   error_message = <<-MESSAGE
-    Error creating backmerge pull request: #{e.message}
-    If this is not the first time you are running the release task, the backmerge PR for the version `#{version_to_merge}` might have already been previously created.
-    Please close any previous backmerge PR for `#{version_to_merge}`, delete the previous merge branch, then run the release task again.
+    Error creating backmerge pull request:
+
+    ```
+    #{e.message}
+    ```
```

Comment:
```suggestion
    #{e.message}
```

One thing I realize happened is that the ``` does something strange, where the entire message (above and below) becomes part of this block. For now in new apps I'm working on (WPiOS/WPAndroid), I've just removed it and let only the line breaks, but perhaps we can think of a better formatting.

Reply:
Removing the code fencing sounds good to me. Hopefully we'll see little of this error, but if it happens often and users found it poorly rendered, we can address then.